### ISSUE TITLE
Permission to translate into Japanese

### ISSUE THREAD

**Issue by Gedevan-Aleksizde** [Litauen/LT_Education#1]
Can I create a Japanese translation?

**Comment by Litauen** [comment]
Yes, you can translate, but you can't release it as a separate 'mod'. 
You can send me the translation and I will include it into the Education mod with full credits to you.

**Comment by Gedevan-Aleksizde** [comment]
I see. I will create it as a pull request later.

**Comment by Gedevan-Aleksizde** [comment]
Hi! Before submitting the Japanese translation, I want you to change some original text. How about it?

https://github.com/Litauen/LT_Education/pull/2
